apps/scan/frontend/src/components/session_time_limit_tracker.tsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
 import { SessionTimeLimitTracker as SessionTimeLimitTrackerBase } from '@votingworks/ui';
 
-import { getAuthStatus, logOut, updateSessionExpiry } from '../api';
+import { getAuthStatus, getConfig, logOut, updateSessionExpiry } from '../api';
 
 export function SessionTimeLimitTracker(): JSX.Element {
   const authStatusQuery = getAuthStatus.useQuery();
   const logOutMutation = logOut.useMutation();
+  const configQuery = getConfig.useQuery();
```

Comment:
I added it to `getConfig` instead of its own query because it seemed consistent within VxScan to return system settings from the single config query. For example, there's no `getElectionDefinition` query; it's just returned from [`getConfig`](https://github.com/votingworks/vxsuite/blob/main/apps/scan/backend/src/app.ts#L130). 

That said, I'm not opposed to breaking up `getConfig` to individual queries. It's just a matter of prioritizing the refactor.

Reply:
I don't feel too strongly either way, with maybe a slight inclination toward consistency. Another downside to the catch-all config object is that there isn't a 1:1 mapping between mutations and the query that they invalidate. Right now, a number of different mutations require invalidating the catch-all config query (a 1:many mapping).

I'll open an issue for this since I don't think it's pressing

Reply:
Yeah I think more fine-grained query/mutation pairings makes more sense!